lib/bootstrap.js

```diff
@@ -110,7 +112,7 @@ function getDefaultTemplateConfig(dir) {
     name: "oghliner-template-app",
     repository: "https://oghliner-template-app.git",
     description: "A template app bootstrapped with oghliner.",
-    license: "Apache-2.0",
+    license: "Apache-2.0"
```

Comment:
Nit: trailing commas in object literals are fairly conventional in this codebase, so I would leave this one.